addons/mrp/static/src/components/bom_overview_control_panel/mrp_bom_overview_control_panel.xml

```diff
@@ -12,7 +12,7 @@
                         <t t-if="props.showVariants">
                             <button t-on-click="() => this.props.print(true)" type="button" class="btn btn-secondary text-nowrap">Print All Variants</button>
                         </t>
-                        <button t-on-click="clickTogglefold" type="button" class="btn btn-secondary"><t t-esc="props.allFolded ? 'Unfold' : 'Fold'"/></button>
+                        <button t-on-click="clickTogglefold" type="button" class="btn btn-secondary" t-esc="foldButtonText"/>
```

Comment:
You could as well have used `<t t-if"props.allFolded">` and `<t t-else="">` child nodes inside the `<button>` node instead of adding a JS getter 🤷‍♂️ But of course both work fine.